pkg/cli/debug/debug.go

```diff
@@ -859,17 +864,6 @@ func (o *DebugOptions) approximatePodTemplateForObject(object runtime.Object) (*
 				},
 			},
 		}, o.NodeName), nil
-	case *imagev1.ImageStreamImage:
-		// create a minimal pod spec that uses the image referenced by the istag without any introspection
-		// it possible that we could someday do a better job introspecting it
-		return setNodeName(&corev1.PodTemplateSpec{
-			Spec: corev1.PodSpec{
-				RestartPolicy: corev1.RestartPolicyNever,
-				Containers: []corev1.Container{
-					{Name: "container-00", Image: t.Image.DockerImageReference},
-				},
-			},
-		}, o.NodeName), nil
```

Comment:
this is a dupe, cleanup

Reply:
It's not dupe, the above is ImageStream**Tag** this is IS**Image**

Reply:
:woman_facepalming: I swear I stared at that/quadruple-checked!